README.md

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright (c) 2007-2022 Louisiana State University                             -->
+<!--                                                                                -->
+<!--   Distributed under the Boost Software License, Version 1.0. (See accompanying -->
+<!--   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)        -->
+
+# HPXC - A C interface to a subset of HPX
+
+## Build
+```sh
+cmake -S . -DHPX_DIR=/path/to/hpx/lib/cmake/HPX -B cmake-build/
+cmake --build cmake-build/ --parallel
+```
+
+> **NOTE:**
+>
+>HPX must be built with `HPX_WITH_DYNAMIC_HPX_MAIN=OFF`.
```

Comment:
I needed this to build the examples, not sure if it's needed in general.

Reply:
Do the examples still run if you disable this option?

Reply:
It won't compile if it's not set to 'OFF'. I get `undefined reference to 'main'`.

With `HPX_WITH_DYNAMIC_HPX_MAIN=OFF`, examples run, but not all successfully. The 'counter' examples hang, and the cancel example not work as expected, printing "something went wrong."

Reply:
Ok, thanks! If some of the examples don't work, then it's probably unrelated to this cmake option.